things/service.go

```diff
@@ -213,6 +243,10 @@ func (ts *thingsService) ViewThing(ctx context.Context, token, id string) (Thing
 		return Thing{}, errors.Wrap(ErrUnauthorizedAccess, err)
 	}
 
+	if err := ts.authorize(ctx, res.GetId(), id, ownerRelationKey, accessRelationKey); err != nil {
```

Comment:
why both owner and access relation?

Reply:
Because I've thought that when a user shares a thing with other users, other users will have an `access` relation on the thing. Therefore, I checked that relation here but actually, this part does not belong to this PR.

Reply:
@buraksekili right, this should be in subsequent PR, and I think that we need to solve that with policy design